web/themes/webspark/renovation/assets/js/renovation.script.js

```diff
@@ -1,2 +1,205 @@
-(()=>{var r,e={63:()=>{},461:()=>{},179:()=>{}},o={};function n(r){var t=o[r];if(void 0!==t)return t.exports;var v=o[r]={exports:{}};return e[r](v,v.exports,n),v.exports}n.m=e,r=[],n.O=(e,o,t,v)=>{if(!o){var a=1/0;for(p=0;p<r.length;p++){for(var[o,t,v]=r[p],i=!0,f=0;f<o.length;f++)(!1&v||a>=v)&&Object.keys(n.O).every((r=>n.O[r](o[f])))?o.splice(f--,1):(i=!1,v<a&&(a=v));if(i){r.splice(p--,1);var l=t();void 0!==l&&(e=l)}}return e}v=v||0;for(var p=r.length;p>0&&r[p-1][2]>v;p--)r[p]=r[p-1];r[p]=[o,t,v]},n.o=(r,e)=>Object.prototype.hasOwnProperty.call(r,e),(()=>{var r={540:0,522:0,63:0};n.O.j=e=>0===r[e];var e=(e,o)=>{var t,v,[a,i,f]=o,l=0;if(a.some((e=>0!==r[e]))){for(t in i)n.o(i,t)&&(n.m[t]=i[t]);if(f)var p=f(n)}for(e&&e(o);l<a.length;l++)v=a[l],n.o(r,v)&&r[v]&&r[v][0](),r[v]=0;return n.O(p)},o=self.webpackChunkrenovation=self.webpackChunkrenovation||[];o.forEach(e.bind(null,0)),o.push=e.bind(null,o.push.bind(o))})(),n.O(void 0,[522,63],(()=>n(63))),n.O(void 0,[522,63],(()=>n(461)));var t=n.O(void 0,[522,63],(()=>n(179)));t=n.O(t)})();
+/******/ (() => { // webpackBootstrap
```

Comment:
Trying to include un-minified JS here.  Needs to be generated using `yarn production`. Please check the compiled CSS, too, to make sure it's minified in the same way.